### DISCUSSION TITLE
Rest API

### DISCUSSION THREAD

**Discussion by Axelgoris99** [pocketbase/pocketbase#2847] [Q&A]
Heeeeeey guys!

Any idea on how the rest api works? 
I'd like to use this as my backend and access it through en ESP32 (Arduino) so in C and there's no SDK available.
I was wondering if it was possible!
Thanks a lot :)
Axel

**Comment by ganigeorgiev** [comment]
You can find general web APIs documentation under the ["Web APIs" section](https://pocketbase.io/docs/api-records/) in the main docs site (_it will be slightly improved in the next release_).
Alternatively, you can also check the available APIs for a specific collection in the _"Admin UI > Collections > {collection} > API Preview"_.

**Reply by Axelgoris99** [reply]
Oh indeed, I'm super blind!!
Sorry for bothering you and thanks for the quick reply :)
(If I have time, I might put on some C SDK, but I'll see if I can manage it!)
